ush/machine/wcoss2.yaml

```diff
@@ -9,9 +9,9 @@ platform:
   MET_BIN_EXEC: bin
   MET_INSTALL_DIR: /apps/ops/para/libs/intel/19.1.3.304/met/10.1.1
   DOMAIN_PREGEN_BASEDIR: /lfs/h2/emc/lam/noscrub/UFS_SRW_App/develop/FV3LAM_pregen
-  QUEUE_DEFAULT: dev
-  QUEUE_FCST: dev
-  QUEUE_HPSS: dev_transfer
+  QUEUE_DEFAULT: devhigh
+  QUEUE_FCST: devhigh
+  QUEUE_HPSS: devhigh
```

Comment:
Is "devhigh" available for other users? If not, let's change them when AQM.v7 is delivered to NCO.